examples/user_interfaces/toolmanager.py

```diff
@@ -83,10 +114,21 @@ def set_lines_visibility(self, state):
 fig.canvas.manager.toolbar.add_tool('zoom', 'foo')
 
 # Remove the forward button
-fig.canvas.manager.toolmanager.remove_tool('forward')
+tool_mgr.remove_tool('forward')
 
 # To add a custom tool to the toolbar at specific location inside
 # the navigation group
 fig.canvas.manager.toolbar.add_tool('Hide', 'navigation', 1)
 
+for i, c in enumerate(['yellowgreen', 'forestgreen']):
+    sidebar = fig.canvas.manager._get_toolbar()
```

Comment:
@fariza Also concerened by the ``_get_toolmanager`` function, it doesn't, it creates a new tool manager and creates it.  Do you foresee the figuremanager having more than one toolmanager?  If not then I don't think it needs a separate function...